examples/proto/Booking.proto

```diff
@@ -6,6 +6,7 @@
  *
  * Author: Elvis Stansvik
  */
+syntax = "proto2";
```

Comment:
I can start learning the code base and see if I can offer some help. This tool is very nice and I really enjoy using it.